cmd/vicadmin/vicadm.go

```diff
@@ -529,9 +547,22 @@ func (s *server) listenPort() int {
 	return s.l.Addr().(*net.TCPAddr).Port
 }
 
+func basicHTTPAuthenticator(handler http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		user, password, ok := r.BasicAuth()
+		if !ok || (user != "root" || password != "thisisinsecure") {
```

Comment:
Its fine to start with using HTTP Basic auth, its just the in-memory hardcoded credentials that should only be used when testing.  Instead of inlining the test credentials here, we should either move this func to _test.go or do something like:
```go
var validateCredentials func(username string, password string) bool
```
And have _test.go set validateCredentials to a func that uses the in memory test credentials.